apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/UrlSignerV4TestData.json

```diff
@@ -121,14 +121,14 @@
         "object": "test-object",
         "headers":
         {
-            "X-Goog-Encryption-Key": [ "ignored" ],
-            "X-Goog-Encryption-Key-Sha256": [ "ignored" ],
-            "X-Goog-Encryption-Algorithm": [ "ignored" ]
+            "X-Goog-Encryption-Key": [ "key" ],
+            "X-Goog-Encryption-Key-Sha256": [ "key-hash" ],
+            "X-Goog-Encryption-Algorithm": [ "AES256" ]
         },
         "method": "GET",
         "expiration": 10,
         "timestamp": "20190201T090000Z",
-        "expectedUrl": "https://storage.googleapis.com/test-bucket/test-object?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=test-iam-credentials%40dummy-project-id.iam.gserviceaccount.com%2F20190201%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20190201T090000Z&X-Goog-Expires=10&X-Goog-SignedHeaders=host&X-Goog-Signature=95e6a13d43a1d1962e667f17397f2b80ac9bdd1669210d5e08e0135df9dff4e56113485dbe429ca2266487b9d1796ebdee2d7cf682a6ef3bb9fbb4c351686fba90d7b621cf1c4eb1fdf126460dd25fa0837dfdde0a9fd98662ce60844c458448fb2b352c203d9969cb74efa4bdb742287744a4f2308afa4af0e0773f55e32e92973619249214b97283b2daa14195244444e33f938138d1e5f561088ce8011f4986dda33a556412594db7c12fc40e1ff3f1bedeb7a42f5bcda0b9567f17f65855f65071fabb88ea12371877f3f77f10e1466fff6ff6973b74a933322ff0949ce357e20abe96c3dd5cfab42c9c83e740a4d32b9e11e146f0eb3404d2e975896f74"
+        "expectedUrl": "https://storage.googleapis.com/test-bucket/test-object?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=test-iam-credentials%40dummy-project-id.iam.gserviceaccount.com%2F20190201%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20190201T090000Z&X-Goog-Expires=10&X-Goog-SignedHeaders=host%3Bx-goog-encryption-algorithm&X-Goog-Signature=5cd533eae44467fcc55ba449228a178e1afb39a72c1e01c3b6c8408451169f01c7af634804bcacc9c437dfbc5c1e8ce82bba4d7a1a654734e790ba3bff036cbbb04f564c4dec91717d6f671bd564096d600b4fcc8663b35a520bc15451bcc529edc9a219f8ba7369d0de66da524e899989260f65825ddf2d6539e21c4a88461c01daa1c9e06bc5b8f827d20f712cd44904b4a316f4a4e85d187f80e114929edd9c30e1cf3ff7f04fb01ac7e75f1c7a22e2f4047f73c57a315ec59089243401f00c41391dc108ffc39fe8fc393a71a4fb899f4178fbbb609dfbe33a5d7f69029f233ea8bdc612a2c59b4d620c2a10a5d5222a51039754581d0b111120fa1b88b1"
```

Comment:
This URL should be updated to include the key-sha256 and key in signedHeaders

Reply:
Now updated.